docs/src/inputs/prompt/examples/exampleAlertPrompt.jsx

```diff
@@ -0,0 +1,46 @@
+import {
+    Button,
+    Prompt,
+} from 'react-cm-ui';
+import React, { useState } from 'react';
+
+function ExampleAlertPrompt() {
+    const [showPrompt, setShowPrompt] = useState(false);
+
+    function onPromptClick() {
+        setShowPrompt(true);
+    }
+
+    function onYesClick() {
+        setShowPrompt(false);
+    }
+
+    function onNoClick() {
+        setShowPrompt(false);
+    }
+
+    return (
+        <div>
+            <Prompt
+                icon="close"
+                inline
+                inlineMessageColor="alert"
+                onClick={() => {}}
+                onNoClick={onNoClick}
+                onYesClick={onYesClick}
+                show={showPrompt}
+            >
+                <Button
+                    color="alert"
+                    disable={showPrompt}
+                    onClick={() => onPromptClick()}
+                    outlined
+                >
+                    Remove
+                </Button>
+            </Prompt>
+        </div>
```

Comment:
This needs all three button states shown as in:

![image](https://user-images.githubusercontent.com/1100790/98308962-2dd1da00-1f7e-11eb-83b6-33894f10f7a7.png)

Reply:
Fixed